lib/helpers.js

```diff
@@ -25,5 +25,8 @@ module.exports = {
 			uri = url.resolve(origin, uri);
 		}
 		return uri;
+	},
+	resolveUriForAuth2(uri, restUrl) {
+		return restUrl + (/\.com(.*)$/g).exec(uri)[1];
```

Comment:
I would prefer using the `string.split(separator)` method instead of a regular expression to increase the readability and minimize the time needed for someone to understand what this method returns. Also, I would add some tests since we already have a spec for the helpers module.